src/pages/Facility/settings/devices/CreateDevice.tsx

```diff
@@ -1,23 +1,33 @@
-import { navigate } from "raviger";
+import { navigate, useQueryParams } from "raviger";
 import { useTranslation } from "react-i18next";
 import { toast } from "sonner";
 
 import { Separator } from "@/components/ui/separator";
 
-import PageTitle from "@/components/Common/PageTitle";
-
 import DeviceForm from "@/pages/Facility/settings/devices/components/DeviceForm";
 
+import DeviceTypeIcon from "./components/DeviceTypeIcon";
+
 interface Props {
   facilityId: string;
 }
 
 export default function CreateDevice({ facilityId }: Props) {
   const { t } = useTranslation();
+  const [qParams] = useQueryParams<{ type?: string }>();
 
   return (
     <div className="space-y-3 max-w-3xl mx-auto">
-      <PageTitle title={t("add_device")} />
+      <div className="inline-flex items-center">
+        <DeviceTypeIcon type={qParams.type} className="size-5 mr-2" />
+        <span className="text-2xl font-bold">
+          {t("add")}{" "}
+          {qParams.type && (
+            <>{qParams.type.charAt(0).toUpperCase() + qParams.type.slice(1)} </>
+          )}
+          {t("device")}
```

Comment:
this is not the right way to handle translations of sentences. word by word translations doesn't make sense in all languages. this would be correct in english, but not necessarily in other languages.

order of words may differ.

refer: https://www.i18next.com/translation-function/interpolation